patches/third_party/catapult/tracing-tracing-metrics-system_health-loading_metric.html.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/tracing/tracing/metrics/system_health/loading_metric.html b/tracing/tracing/metrics/system_health/loading_metric.html
+index 2c5ab4b2d038cafca4201c8445cfe677c93c24c4..d3618406df4c181a92cab73ba4388b0ce9d88923 100644
+--- a/tracing/tracing/metrics/system_health/loading_metric.html
++++ b/tracing/tracing/metrics/system_health/loading_metric.html
+@@ -744,6 +744,7 @@ tr.exportTo('tr.metrics.sh', function() {
+     for (const pid in chromeHelper.rendererHelpers) {
+       const rendererHelper = chromeHelper.rendererHelpers[pid];
+       if (rendererHelper.isChromeTracingUI) continue;
++      let l = rendererHelper.process.labels; if (l !== undefined && l.length === 1 && l[0].startsWith('chrome-extension://')) continue;  // Ignore extension background pages
```

Comment:
why do we want to ignore background pages?

Reply:
Because of Brave extension. That results in recording an extra samples, which breaks the test.
Chromium doesn't have any test with active extensions => we have to fix it ourself.